gprofiler/devkmsg.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+import errno
+import os
+import time
+from typing import List, Tuple
+
+from gprofiler.kernel_messages import Provider
+from gprofiler.log import get_logger_adapter
+
+logger = get_logger_adapter(__name__)
+
+# See linux/printk.h
+CONSOLE_EXT_LOG_MAX = 8192
```

Comment:
I didn't find it in the kernel btw. Which version did u find it at?

Reply:
Since v4.2-rc1:: d43ff430f434d862db59582c0f1f02382a678036